### ISSUE TITLE
GHSA entry reports incorrect scoring system for 2024 vulnerabilities

### ISSUE THREAD

**Issue by mjherzog** [aboutcode-org/vulnerablecode#1644]
While reviewing recent (2024) GHSA data in VCIO I found that VulnerableCode does not seem to recognize where GHSA is using CVSS v4 instead of CVSS 3.1 for scoring. Some examples are:

- https://github.com/advisories/GHSA-735f-pc8j-v9w8 for VCID-7ffu-u28q-bqgw
- https://github.com/advisories/GHSA-78wr-2p64-hpwj for VCID-bbcg-ujrw-77gc
- https://github.com/advisories/GHSA-wm9w-rjj3-j356 for VCID-yktk-48uz-aaac

In these cases VCIO reports: cvssv3.1_qr =HIGH for the GHSA. It should report a the Severity System as cvss4 or similar. I am not sure what the "_qr" suffix means in "cvssv3.1_qr".

**Comment by DennisClark** [comment]
In general I think that we always collect CVSS v3.x scores. Perhaps the bug here is that we are not collecting the "ADP: CISA-ADP" score when there is no "NVD" score.  See screenshot.

<img width="1034" alt="CVE-2024-34750 Detail 2024-11-11 at 12 59 51" src="https://github.com/user-attachments/assets/4e013cc5-d0f5-4db9-a7a2-2a5d77b7c673">

**Comment by mjherzog** [comment]
Good point - I missed that possibility but in any case we need to document/qualify what score we are collecting.
And we need to prepare for / starting collecting v4.0 scores.
